.github/workflows/beekeeper.yml

```diff
@@ -13,7 +13,7 @@ env:
   RUN_TYPE: "PR RUN"
```

Comment:
title of this PR might be more suitable for `feat` label then `chore` considering that this is not trivial upgrade